src/human_aware_rl/ppo/ppo_rllib_from_params_client.py

```diff
@@ -391,7 +391,13 @@ def _env_creater(env_config):
 
 
 def run(params):
+    layout = "MyPPOModel"
```

Comment:
Is this the right variable name? Doesn't seem like a layout?

Reply:
Ah yeah I was going to bring this up during the meeting today. So it looks like ex_fp (the experiment from ppo_rllib_from_params_client.py) supports creating a MDP not from a specific layout but from a "mdp_params_schedule_fn". So I just tried to get a placeholder for the layout name, so it is grouped correctly in wandb. It is not actually a layout and I should have probably picked a better name.

Reply:
Probably this is not relevant after we've dealt with the case below 😄